packages/server/assets/automations/typedefs/automation.graphql

```diff
@@ -1,112 +1,118 @@
-extend type Model {
-  automationStatus: AutomationsStatus
+enum AutomationRunStatus {
```

Comment:
- Moved types around for improved readability
- Renamed `runStatus` to `status`, the `run` prefix is redundant, renamed `functionRunStatuses` to `functionRuns` so that input and output types are more alike and simpler
- Renamed types a bit (e.g. removed `Model` prefix, cause the types are not necessarily Model specific)
- Removed some unnecessary/commented out typedefs
- Added webhookId support